.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: node_js
+node_js:
+    - "4.4"
+install: 
+    - npm run init
+    - npm install
```

Comment:
With my other pending PR https://github.com/Microsoft/PowerBI-Node/pull/9 this order should definitely change